### PR TITLE
Added customfileds capability

### PR DESCRIPTION
**Pull request by p0pe1e**
Added capability to get cutom fields when ${incident.<customfiledname>} is in the temaplge and arguemnt to replace not found fields with empty strings

fixes https://github.com/demisto/etc/issues/12783

### CODE REVIEW

File: Scripts/script-CreateEmailHtmlBody.yml [hunk 1]
```diff
@@ -83,7 +95,17 @@ args:
...
+  - "no"
+  description: if true, replace path not found with empty string, otherwise leave
+    as is
+  defaultValue: "no"
 outputs:
 - contextPath: htmlBody
   description: HTML Body
 scripttarget: 0
+runonce: false
+releaseNotes: added ability to get custom fileds when the path is incident.<customFieldName>, and added an arugment to replace not found values with empty string
```
Comment (gshriki) [inline Scripts/script-CreateEmailHtmlBody.yml]: would rephrase to something like:
Added the ability to have custom fields in the template in both .incident.CustomFields.<fieldname> and incident.<fieldname> formats. Also, added the option to replace non-found placeholder values with  empty string

File: Scripts/script-CreateEmailHtmlBody.yml [hunk 2]
```diff
@@ -83,7 +95,17 @@ args:
 - name: object
   description: Values object provided as stringified JSON
   isArray: true
+- name: replaceNotFound
```
Comment (gshriki) [inline Scripts/script-CreateEmailHtmlBody.yml]: maybe rename to removeNotFound? replace gives the perception you will allow replacing it with something, where in fact you are just deleting it, ni?

File: Scripts/script-CreateEmailHtmlBody.yml [hunk 3]
```diff
@@ -11,15 +11,15 @@ script: |
   // get a specific label from incident labels
   var getLabel = function(incident,path) {
       label = path.split('.')[2];
-      for (var i=0;i<incident.labels.length;i++) {
-          if (incident.labels[i].type==label) {
+      for (var i=0; i<incident.labels.length; i++) {
```
Comment (gshriki) [inline Scripts/script-CreateEmailHtmlBody.yml]: understand the urge to fix formatting, but this makes the reviews a bit more challenging (also for customers who might want to review diff)... recommend next time avoiding :)

**Comment by gshriki** [comment]
@ori - status?

**Comment by anara123** [comment]
@p0pe1e status?

**Comment by Itay4** [comment]
@bakatzir What is the status of this?

**Comment by bakatzir** [comment]
@Itay4  I merged from master and fixed CR, don't know why i can't see the tests status.
should I open a new PR and close this one?

**Comment by Itay4** [comment]
@bakatzir You cannot see tests status because this branch is not of the Demisto content repo.
You should create a new branch on Demisto content repo, rebase this branch to your branch, merge it and then open a new PR which will trigger CircleCI.
